appveyor.yml

```diff
@@ -74,25 +74,12 @@ build_script:
   - mkdir build
   - cd build
 
-  # build/test without OpenMP, with CFP
-  - if "%COMPILER%"=="msvc" cmake -G "%GENERATOR%" -DCMAKE_BUILD_TYPE="%BUILD_TYPE%" -DZFP_WITH_OPENMP=OFF -DBUILD_CFP=ON ..
-  - if not "%COMPILER%"=="msvc" cmake -G "%GENERATOR%" -DCMAKE_BUILD_TYPE="%BUILD_TYPE%" -DCMAKE_SH=CMAKE_SH-NOTFOUND -DZFP_WITH_OPENMP=OFF -DBUILD_CFP=ON ..
-
-  - if "%COMPILER%"=="msvc" cmake --build . --config "%BUILD_TYPE%"
-  - if not "%COMPILER%"=="msvc" cmake --build .
-
-  - ctest -V -C "%BUILD_TYPE%"
+  # build/test without OpenMP, with CFP (and custom namespace)
+  - if "%COMPILER%"=="msvc" ctest -V -C "%BUILD_TYPE%" -DGENERATOR="%GENERATOR%" -DBUILD_CFP=ON -DCFP_NAMESPACE=cfp2 -DBUILD_OPENMP=OFF -DBUILD_CUDA=OFF -S %APPVEYOR_BUILD_FOLDER%/cmake/appveyor.cmake
+  - if not "%COMPILER%"=="msvc" ctest -V -C "%BUILD_TYPE%" -DGENERATOR="%GENERATOR%" -DBUILD_CFP=ON -DCFP_NAMESPACE=cfp2 -DBUILD_OPENMP=OFF -DBUILD_CUDA=OFF -S %APPVEYOR_BUILD_FOLDER%/cmake/appveyor.cmake
 
   - rm -rf ./*
 
-  # build/test with OpenMP, with CFP custom namespace
-  - if "%COMPILER%"=="msvc" cmake -G "%GENERATOR%" -DCMAKE_BUILD_TYPE="%BUILD_TYPE%" -DZFP_ENDTOEND_TESTS_ONLY=1 -DBUILD_CFP=ON -DCFP_NAMESPACE=cfp2 ..
-  - if not "%COMPILER%"=="msvc" cmake -G "%GENERATOR%" -DCMAKE_BUILD_TYPE="%BUILD_TYPE%" -DCMAKE_SH=CMAKE_SH-NOTFOUND -DZFP_ENDTOEND_TESTS_ONLY=1 -DBUILD_CFP=ON -DCFP_NAMESPACE=cfp2 ..
-
-  - if "%COMPILER%"=="msvc" cmake --build . --config "%BUILD_TYPE%"
-  - if not "%COMPILER%"=="msvc" cmake --build .
-
-  # only run tests not run in previous build, due to appveyor time limit (1 hour)
-  - ctest -V -C "%BUILD_TYPE%" -R ".*Omp.*"
```

Comment:
For the 2nd build, the regex seems to have been dropped. Can we restore this behavior? All tests are run for the 1st build, but for the 2nd build, only run tests matching regex ".*Omp.*"

Reply:
I will look into how we can add this in.

Reply:
Okay updated to only run the OpenMP tests for the second build